Exiled.API/Features/Items/Jailbird.cs

```diff
@@ -110,7 +110,10 @@ public JailbirdWearState WearState
         /// <summary>
         /// Breaks the Jailbird.
         /// </summary>
-        public void Break() => WearState = JailbirdWearState.Broken;
+        public void Break()
+        {
+            Base.SendRpc(JailbirdMessageType.Broken);
```

Comment:
Keep the lambda: `public void Break() => Base.SendRpc(JailbirdMessageType.Broken);`

Reply:
actually i don't even see how the Item will be destroy it's just send Rpc to client